src/xunit.netcore.extensions/TargetFrameworkMoniker.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    [Flags]
+    public enum TargetFrameworkMoniker
+    {
+        Net45 = 1,
+        Net451 = 2,
+        Net452 = 4,
+        Net46 = 8,
+        Net461 = 16,
+        Net462 = 32,
+        Net463 = 64,
```

Comment:
Could we restructure these values so that there's a value that covers all of desktop, a value that covers all of netcoreapp, etc.?  Otherwise each time a new version comes out we'd likely need to back and update all of the corresponding attributions.
e.g.
```C#
[Flags]
public enum TargetFrameworkMoniker
{
    NetFramework = 0x1,
    NetCoreApp = 0x2,
    NetCore50 = 0x4,
    
    NetCore45 = 0x100 | NetFramework,
    NetCore451 = 0x200 | NetFramework,
    NetCore453 = 0x400 | NetFramework,
    ...
}
```